notebooks/official/gen_ai/batch_eval_llm.ipynb

```diff
@@ -0,0 +1,636 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.            "large_model_reference": "BISON",  # see Table 1 for valid values.
</pre><p>"BISON" is not listed in Table 1. Should it be in Table 1? Or should this be <code>text-bison@001</code> instead? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Its a workaround. They still need to be update to backend service (temp)